File: BE-FindingJob/src/main/java/com/example/befindingjob/model/Resume.java
package com.example.befindingjob.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "resumes")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Resume {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer resumeId;

    @OneToOne
    @JoinColumn(name = "application_id", nullable = false, unique = true)
    private Application application;

    private String resumeUrl;

    private java.time.LocalDateTime createdAt;
    private java.time.LocalDateTime updatedAt;
}

File: JobHub/app/src/main/java/com/example/jobhub/entity/Resume.kt
package com.example.jobhub.entity

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import java.time.LocalDateTime

@Parcelize
data class Resume(
    val resumeId: Int,
    val resumeUrl: String?,
    val createdAt: LocalDateTime?,
    val updatedAt: LocalDateTime?,
    val application: Application?
) : Parcelable